interface/main/tabs/menu/menus/standard.json

```diff
@@ -692,7 +692,7 @@
       {
         "label": "Lab Documents",
         "menu_id": "dld0",
-        "target": "pat",
```

Comment:
does doing this type of stuff risk leaving tabs around that are directed to the wrong patient (for example, if close a patient and then open another patient's records)?

Reply:
oh maybe, i'll revert all but the elec reports since it's annoying to have that overtake whatever pt tab it was and test that

Reply:
should be good now @bradymiller but moved to needs review just in case